frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -62,101 +64,142 @@ export default function PersonalInformationIndex() {
-            <p>
-              <InlineLink id="change-preferred-language-button" routeId="$lang+/_protected+/personal-information+/preferred-language+/edit" params={params}>
-                {t('personal-information:index.change-preferred-language')}
-              </InlineLink>
-            </p>
-          )}
-        </DescriptionListItem>
-        <DescriptionListItem term={t('personal-information:index.home-address')}>
-          {personalInformation.homeAddress ? (
-            <Address
-              address={personalInformation.homeAddress.streetName ?? ''}
-              city={personalInformation.homeAddress.cityName ?? ''}
-              provinceState={regionList.find((region) => region.provinceTerritoryStateId === personalInformation.homeAddress!.provinceTerritoryStateId)?.abbr}
-              postalZipCode={personalInformation.homeAddress.postalCode}
-              country={countryList.find((country) => country.countryId === personalInformation.homeAddress!.countryId)?.[i18n.language === 'fr' ? 'nameFr' : 'nameEn'] ?? ' '}
-            />
-          ) : (
-            <p>{t('personal-information:index.no-address-on-file')}</p>
-          )}
-          {useFeature('edit-personal-info') && (
-            <p>
-              <InlineLink id="change-home-address-button" routeId="$lang+/_protected+/personal-information+/home-address+/edit" params={params}>
-                {t('personal-information:index.change-home-address')}
-              </InlineLink>
-            </p>
-          )}
-        </DescriptionListItem>
-        <DescriptionListItem term={t('personal-information:index.mailing-address')}>
-          {personalInformation.mailingAddress ? (
-            <Address
-              address={personalInformation.mailingAddress.streetName ?? ''}
-              city={personalInformation.mailingAddress.cityName ?? ''}
-              provinceState={regionList.find((region) => region.provinceTerritoryStateId === personalInformation.mailingAddress!.provinceTerritoryStateId)?.abbr}
-              postalZipCode={personalInformation.mailingAddress.postalCode}
-              country={countryList.find((country) => country.countryId === personalInformation.mailingAddress!.countryId)?.[i18n.language === 'fr' ? 'nameFr' : 'nameEn'] ?? ''}
-            />
-          ) : (
-            <p>{t('personal-information:index.no-address-on-file')}</p>
-          )}
-          {useFeature('edit-personal-info') && (
-            <p>
-              <InlineLink id="change-mailing-address-button" routeId="$lang+/_protected+/personal-information+/mailing-address+/edit" params={params}>
-                {t('personal-information:index.change-mailing-address')}
-              </InlineLink>
-            </p>
-          )}
-        </DescriptionListItem>
-        <DescriptionListItem term={t('personal-information:index.phone-number')}>
+        <DescriptionListItemTwoColumn term={t('personal-information:index.addresses')}>
```

Comment:
Wasn't sure where to put this, but FYI I made a comment on the PR as a whole:

https://github.com/DTS-STN/canadian-dental-care-plan/pull/1159#pullrequestreview-2017486483

Reply:
Right, I increased the spacing in an attempt to address it.

Went from `space-y-6` to `space-y-7`

Reply:
I still feel there's not enough spacing between the link and heading:
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/9bf12a9a-555e-4db9-abc4-20fc49c0d145)

Removing the `sm` breakpoint prefix from `sm:py-6` on the `DescriptionListItem`/`DescriptionListItemTwoColumn` components (and switching back to `space-y-6`):
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/fa4d3cb0-8782-49b0-bc55-8f6936cdc46d)

Reply:
That works, now implemented in commit 786957903cac1c78cd14fc504ca33214a048bcee.

Reply:
Issue still exists

https://github.com/DTS-STN/canadian-dental-care-plan/commit/786957903cac1c78cd14fc504ca33214a048bcee removes the `space-y-6` but still keeps `sm:py-6` (which should just `py-6`)

Reply:
Whoops, thanks. `sm:py-6` has now been replaced with `py-6`